pkgs/tools/misc/vdirsyncer/default.nix

```diff
@@ -20,6 +20,10 @@ pythonPackages.buildPythonApplication rec {
     requests
     requests_oauthlib # required for google oauth sync
     atomicwrites
+    milksnake
+
+    pkgs.cargo
+    pkgs.rustc
```

Comment:
Please import cargo and rustc without relying on `pkgs`. This is needed for `vdirsyncer.override {}` to work.